dgv/monitoring/hvd/task_functions.py

```diff
@@ -428,6 +429,20 @@ def build_df_for_grist():
 
 
 def update_grist(ti):
+    def update_quality(dataset_id: str):
+        r = requests.get(
+            f"https://www.data.gouv.fr/api/1//datasets/{dataset_id}/",

```

Comment:
Could we use api 2 in this case?
```suggestion
            f"https://www.data.gouv.fr/api/1/datasets/{dataset_id}/",

```